.github/workflows/ci.yml

```diff
@@ -65,7 +69,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Install dependencies
-      run: sudo apt-get install -y valgrind $TEST_DEPENDENCIES
+      run: |
+        sudo apt-get update
```

Comment:
There are two other `apt-get install` commands below.  Do they need `apt-get update` too?

Also, this needs to be fixed in `fscrypt` too, right?

Reply:
So the issue only seems to occur for certain packages with Github CI (the `gcc-multilib` gave me problems on another project). I initially only added it for failing steps, but we should probably do it for all of them.

Reply:
Added to all of them.